packages/attestation-service/config/database.json

```diff
@@ -0,0 +1,11 @@
+{
+  "development": {
```

Comment:
Should this be `integration` to match the env breakdown from other services.
Also should these be 1:1 with our actual testnets (alfajores, pilot)

Reply:
So I believe this is a `sequelize` thing that more denotes the type of environment. I just added this for now. Do you mind if we tackle this in a future PR as actually setting this up for deployment is supposed to be done in a separate ticket #1076